.gitlab-ci.yml

```diff
@@ -322,26 +305,7 @@ lint:
     OPAM_SWITCH: "edge"
     OPAM_VARIANT: "+flambda"
 
-pkg:opam-native:
-  stage: build
-  except:
-    variables:
-      - $ONLY_WINDOWS == "true"
-  interruptible: true
-  # OPAM will build out-of-tree so no point in importing artifacts
-  script:
-    - opam install coq-native
-    - opam pin add --locked=docker --kind=path coq.dev .
-    # Check that native works
-    - echo "Definition f x := x + x." > test_native.v
-    - coqc -native-compiler yes test_native.v
-    - test -f .coq-native/Ntest_native.cmxs
```

Comment:
Please keep this test, otherwise the CI may fail to identify failures with coq-native.

Reply:
Done.